Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -159,6 +159,9 @@ void ControllerInterface::RefreshDevices(RefreshReason reason)
   // with their own PlatformPopulateDevices().
   // This means that devices might end up in different order, unless we override their priority.
   // It also means they might appear as "disconnected" in the Qt UI for a tiny bit of time.
+  // This helps the emulation and host thread to not stall when repopulating devices for any reason.
+  // Every platform that adds a device that is meant to be used as default device should try to not
+  // do it async, to not risk the emulated controllers defaulting not finding a default device.
```

Comment:
"defaulting not finding a default device" doesn't make sense grammatically, but with this one I at least do understand what the comment is supposed to mean.

Reply:
Ah, I see now, you meant "emulated controllers' defaulting". A little confusing using the word "defaulting" like a noun, but I suppose it works.